tests/graphql_spec.rs

```diff
@@ -151,7 +168,15 @@ impl GraphQLSpec {
       if path.is_file() && path.extension().unwrap_or_default() == "graphql" {
         let contents = fs::read_to_string(path.clone())?;
         let path_buf = path.clone();
-        files.push(GraphQLSpec::new(path_buf, contents.as_str()));
+        let spec = GraphQLSpec::new(path_buf, contents.as_str());
+
+        match spec.annotation {
```

Comment:
Also if no annotation is given all tests should run as it is. Ideally you should create two vectors one for only one for all, if only is empty run all if only is non empty run only